css/style.css

```diff
@@ -0,0 +1,285 @@
+ position: relative;
+}
+
+.header-date,
+.header-btn {
+ padding-top: 30px;
+}
+
+header svg {
+ transform: scale(2.3);
+}
+
+.login {
+ font-size: 16px;
+ padding-right: 16px;
+ font-weight: bold;
+}
+
+.subscribe {
+ border-radius: 15px;
+ padding: 8px;
+ background-color: #ffe400;
+ margin-right: 30px;
+}
+
+.hr-icon-bottom {
+ position: absolute;
+ top: 100%;
+ right: 0;
+ content: "";
+ display: block;
+ width: 92px;
+ height: 14px;
+ background-color: #f72210;
+ border-left: 22px solid white;
+ border-top: 16px solid #f72210;
+}
+
+.header-main-2 img,
+.header-main-3 img {
+ width: 50px;
+ height: 50px;
+}
+
+.header-top svg {
+ display: inline;
+ width: 236px;
+ height: 36px;
+}
+.main2-svg-link {
+ margin-top: 0 !important;
+ width: 100px !important;
+ height: 20px;
+ padding: 0 !important;
+}
+
+.main2-svg {
+ margin: 15px 0 0 -90px !important;
+ width: 300px !important;
+}
+
+.header-main-2 svg {
+ margin-left: -30px;
+ height: 40px;
+ width: 350px;
+ margin-top: 25px;
+ position: static;
+}
+
+.header-main-3 svg {
+ margin-left: 30px;
+ height: 20px;
+ width: 80px;
+ margin-top: 0;
+}
+
+
+.header-bottom-menu li a {
+ font-size: 14px;
+}
+
+.header-top-menu li a {
+ font-size: 14px;
+}
+
+.header-top svg {
+ margin-top: 20px;
+ fill: white;
+}
+
+.fa {
+ font-size: 14px;
+ text-decoration: none;
+ background: rgb(161, 138, 138);
+}
+
+.copy-right {
+ position: absolute;
+ font-size: 10px;
+ bottom: 10px;
+ left: 0;
+}
+
+.fa-user {
+ display: none;
+}
+
+@media screen and (max-width: 992px) {
+ .header-main-1,
+ .header-top-menu,
+ .header-bottom-menu {
+   display: none;
+}
+
+.container-fluid {
+   padding: 0;
+   margin: 0;
+}
+
+.fa-user {
+   display: none;
+}
+
+.header-main-2 svg {
+   transform: scale(1.05);
+   padding-left: 80px;
+ }
+}
+
+hr {
+  background-color: #212529;
+}
+.footer {
+padding-left: 30px;
+}
+
+.footer-logo-large {
+  color: #f72210;
+  font-size: 23px;
+  font-weight: bold;
+}
+
+.footer-logo-legal {
+  font-size: 12px;
+}
+
+.footer-eds {
+  color: grey;
+}    
+
+.footer-down {
+  border-right: 1px solid rgba(0, 0, 0, 0.2);
+  padding-right: 10px;
+  padding-left: 10px;
+}
+
+.footer-links {
+  font-size: 14px;
+  color: grey;
+}
+@media screen and (min-width: 992.99px) {
+ .header-main-2 {
+   display: none;
+ }
+
+.header-main-1-lg > div {
+   padding-top: 40px;
+ }
+}
+
+@media screen and (max-width: 767.99px) {
+ .header-main-2 svg {
+   margin-left: -30px;
+   height: 30px;
+   width: 250px;
+   margin-top: 25px;
+   position: static;
+ }
+
+.header-main-3 svg {
+   margin-bottom: 10px;
+ }
+
+.header-main-3 .person-icon-acc {
+   margin-top: 10px !important;
+ }
+
+.header-main-2 img,
+.header-main-3 img {
+   width: 40px;
+   height: 40px;
+ }
+
+.login {
+   display: none;
+}
+
+.subscribe {
+   border-radius: 15px;
+   padding: 5px;
+   background-color: #ffe400;
+   margin-right: 5px;
+   margin-left: 10px;
+   font-size: 12px;
+}
+
+ .social-icons {
+   float: left !important;
+ }
```

Comment:
- [ ] Try not to use `!important ` if it can be avoided. This can be done with the bootstrap class `float-md-left`